### ISSUE TITLE
Custom recording directory

### ISSUE THREAD

**Issue by yephny** [the-weird-aquarian/OpenRec#4]
OpenRec currently does not support changing the directory where recordings are stored. I cannot view my recordings in `sdcard/VoiceRecorder` with the app.

**Comment by StellarSand** [comment]
Yes, all recordings are currently stored in `Android/data/com.openrec/files/records`. We didn't want to mess around with Android permissions for external storage at the time of developing this app. With that being said, there's things that need to be implemented - recording formats like .wav, custom recording directories, more languages and a few other things. Development hasn't been super active on this app due to personal reasons but we are looking forward to fix stuff soon.

**Comment by yephny** [comment]
Thanks, it's great to see what you plan to implement in the future, I will not have to open an issue for them.
